src/dialog/style.ts

```diff
@@ -58,6 +59,8 @@ export const DIALOG_CSS = `
     color: var(--kuc-dialog-header-color);
     padding: 0 24px;
     align-self: center;
+    overflow-wrap: break-word;
+    max-width: 492px;
```

Comment:
Do not need to set the max-width for title.
Using "overflow-wrap: anywhere; word-break: normal;" will be better.